GVFS/GVFS.Common/NativeMethods.cs

```diff
@@ -98,14 +98,6 @@ public static void MoveFile(string existingFileName, string newFileName, MoveFil
             }
         }
 
-        public static void CreateHardLink(string newLinkFileName, string existingFileName)

```

Comment:
I saw this was removed, did we not need it or is it unsupported?  I didn't see a HardLink issues, but I'm known to be blind!

Reply:
@jeschu1 this is dead code.

I am guessing it was added when we were experimenting with using hardlinks to create GVFS_projection rather than performing a file copy.

cc: @kewillford as FYI that I'm removing this code.

Reply:
Yeah we can always add it back in if we want to try and use hard links again.